exec/java-exec/src/main/resources/rest/query/result.ftl

```diff
@@ -30,6 +30,22 @@
   <a href="/queries">back</a><br/>
   <div class="page-header">
   </div>
+  <div>
+  <table><tr>
+    <td align='left'>
+      <button type="button"  title="Open in new window" onclick="popOutProfile('${model.getQueryId()}');" class="btn btn-default btn-sm">
+      <b>Query Profile:</b> ${model.getQueryId()} <span class="glyphicon glyphicon-new-window"/></button>
+     </td><td align="right" width="100%">
+       <div class="input-group">
+         <span class="input-group-addon" style="font-size:95%">Delimiter </span>
+         <input id="delimitBy" type="text" class="form-control input-sm" name="delimitBy" title="Specify delimiter" placeholder="Required" maxlength="2" size="2" value=",">
+       </div></td><td>
+       <button type="button"  title="Export visible table as CSV" onclick="exportTableAsCsv('${model.getQueryId()}');" class="btn btn-default btn-sm">
```

Comment:
Maybe we can add tooltip that explains that we export only visible table? If user needs to export all results, then should chose all rows to display?